src/soter/openssl/soter_rsa_key_3.c

```diff
@@ -123,8 +125,10 @@ soter_status_t soter_engine_specific_to_rsa_priv_key(const soter_engine_specific
     int rsa_mod_size;
     size_t output_length;
     uint32_t* pub_exp;
-    // This bigint is reused multiple times: read into it, serialize it, read into it...
-    BIGNUM* tmp = NULL;
+    // Maximum supported RSA key is 8192 bits (1024 bytes)
+    unsigned char bigint_buf[1024];
```

Comment:
Maybe we should use the named constants? :) I've seen similar comments for similar places in the previous PRs, but I don't know the general style we use in Themis.

Reply:
Since it is used for multiple purposes, I had no better idea about naming

Reply:
lets use `#define RSA_KEY_BYTES_MAX = RSA_BYTE_SIZE(8192)` ? in the src/sother/soter_rsa_key.h. I think we will need it in the other parts in the future, for example for boringssl